server/controllers/work.js

```diff
@@ -15,3 +18,45 @@ export const work = async(ctx, next) => {
 
   return next();
 };
+
+function getResultsWithImages(results) {
+  if (!results) return;
+
+  if (results.error) {
+    // Display some error messaging to the user here?
+    console.error(results.error);
+
+    return;
+  }
+
+  return results.results.map((result) => {
+    const miroId = result.identifiers[0].value;
+    const cleanedMiroId = miroId.match(/(^\w{1}[0-9]*)+/g, '')[0];
+    const miroFolder = `${cleanedMiroId.slice(0, -3)}000`;
+    const imgLink = `http://s3-eu-west-1.amazonaws.com/miro-images-public/${miroFolder}/${miroId}.jpg`;
+    return Object.assign({}, result, {imgLink});
+  });
+}
+
+export const search = async (ctx, next) => {
+  const { query, page } = ctx.query;
+  const results = query && query.trim() !== '' ? await getWorks(query, page) : null;
+  const resultsWithImages = getResultsWithImages(results);
+  const pageSize = results && results.pageSize ? results.pageSize : null;
```

Comment:
It will assign to the last evaluated equation - it's used in JSX templates a lot:
https://github.com/wellcometrust/wellcomecollection.org/pull/740/files#diff-51ce732feab7de04efc1f739b5bdef3aR17